### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that conversion of shared mailbox can be done on-prem (cmdlets can be only used Starting in Cumulative Update 21 for Exchange Server 2013 and Cumulative Update 10 for Exchange Server 2016.)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3590

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
 
 ## Convert a user's mailbox in a hybrid environment
 
+> [!NOTE]
+> Starting in Cumulative Update 21 for Exchange Server 2013 and Cumulative Update 10 for Exchange Server 2016 in an on-premises Exchange Server environment, you can directly create or modify a remote shared mailbox. For more information, View [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/help/4133605/cmdlets-to-create-modify-remote-shared-mailbox-in-on-premises-exchange)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
> Starting in Cumulative Update 21 for Exchange Server 2013 and Cumulative Update 10 for Exchange Server 2016 in an on-premises Exchange Server environment, you can directly create or modify a remote shared mailbox. For more information, see [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/help/4133605/cmdlets-to-create-modify-remote-shared-mailbox-in-on-premises-exchange).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SKjerland Please merge this PR. Thanks.

**Comment by SKjerland** [comment]
@yogkumgit or @JohanFreelancer9 - Please resolve conflicts and then ping me again so I can merge. Thanks!

**Comment by JohanFreelancer9** [comment]
@msbemba - would you be able to resolve the conflicts, please.

**Comment by msbemba** [comment]
The conflicts have been resolved. Thank you. @SKjerland

**Pull request merged**